src/js/app.js

```diff
@@ -3,9 +3,9 @@
  *
  * This is where you write your app.
  */
-
-var UI = require('ui');
-var Vector2 = require('vector2');
+require('./main');
+var UI = require('./ui');
+var Vector2 = require('./lib/vector2');
```

Comment:
I hope users will be okay with having to add './' since now their apps won't work if recompiled. I'll have to remember to update the docs too.